### PR TITLE
Deitsches Keiser Schmarrn Rezept

### PR DESCRIPTION
**Pull request by mto078**
Hmmmmmm!!!!!!!

### CODE REVIEW

File: KaiserSchmarrn.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Zutaten:
+200 g Mehl
```
Comment (pst019) [inline KaiserSchmarrn.txt:2]: + 200g Roggenmehl

File: KaiserSchmarrn.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+30 g Zucker
+1 Prise Salz
+4 Stk Eier
```
Comment (pst019) [inline KaiserSchmarrn.txt:5]: + 4 Stk Bioeier

**Review by pst019 — COMMENTED** [review]
Looks good, beside small changes.

File: KaiserSchmarrn.txt (hunk 1 above)
Comment (mto078) [reply KaiserSchmarrn.txt:2]: Forget it!!

File: KaiserSchmarrn.txt (hunk 2 above)
Comment (mto078) [reply KaiserSchmarrn.txt:5]: ok

**Comment by Hegebf** [comment]
Please make this so we can see what it is :)

**Pull request merged**